zerver/lib/actions.py

```diff
@@ -4961,6 +4961,20 @@ def do_change_stream_message_retention_days(
     send_event(stream.realm, event, can_access_stream_user_ids(stream))
 
 
+def set_education_realm_permissions(realm: Realm) -> None:
```

Comment:
One more improvement we could do here is to rename this function `set_realm_permissions_based_on_org_type`, then this can be more of a generic function where we can add more org types in the future.